sections/header.liquid

```diff
@@ -729,7 +729,7 @@
     "@type": "Organization",
     "name": {{ shop.name | json }},
     {% if section.settings.logo %}
-      "logo": {{ section.settings.logo | image_url: width: section.settings.logo.width | prepend: "https:" | json }},
+      "logo": {{ section.settings.logo | image_url: width: 700 | prepend: "https:" | json }},
```

Comment:
Just out of curiosity, where is this requirement for the logo width coming from?